src/client/reducers/audioReducer.js

```diff
@@ -27,40 +27,42 @@ const initialState = {
 export default function (state=initialState, action){
     
     switch(action.type){
-        case CREATE_AUDIO_DATA: {  
-            const {name: trackName, type: trackType, size: trackSize, file: audioFile, audio: audioFromFile, source: audioFromFileSource} = action.payload;                                            
+        case CREATE_AUDIO_DATA:{  
+            let {trackName, trackSize, trackType, audioFile, audioFromFile, audioFromFileSource} = action.payload;                                                        
             return {
                 ...state,
                 trackName,
                 trackType,
                 trackSize,
                 audioFile,
                 audioFromFile,
-                audioFromFileSource
-                };
+                audioFromFileSource 
             }
+            };            
         case PLAY_PAUSE_SOUND_FROM_FILE:
             return {
                 ...state,
                 playPauseState: !state.playPauseState
             };
         case CREATE_STREAME_DATA: {
-            const {audioLineIn: audioStream, sourceStream: streamSource} = action.payload;
+            let {audioLineIn: audioLineIn, sourceStream: streamSource} = action.payload;
             return {
                 ...state,
-                audioStream,
-                streamSource
+                audioStream: audioLineIn,
+                streamSource: streamSource
                 };
             }     
-        case START_MUTE_STREAME_AUDIO:
+        case START_MUTE_STREAME_AUDIO:             
                 return {
                     ...state,
-                    startMuteState: !(state.startMuteState)
+                    startMuteState: !state.startMuteState                
                 };    
-        case MERGE_CANVAS_WIDTH:
+        case MERGE_CANVAS_WIDTH: {
+            let widthCanvas = parseInt(action.payload, 10)
```

Comment:
use const here and no need to add number system, because its 10 by default

Reply:
ok